lib/icalendar/util/kv.ex

```diff
@@ -4,33 +4,49 @@ defmodule ICalendar.Util.KV do
   """
 
   @doc ~S"""
-    iex> ICalendar.Util.KV.build("foo", "bar")
+  Convert a key and value to an iCal line:
+
+    iex> ICalendar.Util.KV.build("foo", "bar", "bar")
```

Comment:
Passing in bar twice here is a little unclear- what is it supposed to represent? It's just discarded so why do we pass it in?